README.md

```diff
@@ -46,7 +46,7 @@ downloadEmailAttachments({
   log: console.log,
   since: '2015-01-12',
   lastSyncIds: ['234', '234', '5345'] // ids already dowloaded and ignored, helpful because since is only supporting dates without time
-  attachmentHandler: function (attachmentData, callback) {
+  attachmentHandler: function (attachmentData, callback, errorCB) {
```

Comment:
Why don't you use the classic node style `callback(error, response)` here and instead add an extra `errorCB`?